client/src/pages/Profile.js

```diff
@@ -168,6 +174,47 @@ const Profile = ({
   const getActorQuery = () => {
     return organisationId ? `&actorId=${organisationId}` : "";
   };
+  const buildNavMenu = () => {
```

Comment:
With the navigation functionality built in Profile may hard to reuse in the future because every time we want to have a component that supports similar navigation then we will have to rewrite the logic for navigating the correct view and position the view in the right location. I think we should create a new component that's based on Menu, but add this navigation functionality so that it's more reusable (just a suggestion, that's fine if you disagree).

Reply:
menu already is a component all by itself, I didn't make a new menu component All I've made is add a wrapper for menu that styles it according to our profile design. This is how the menu was designed on the feed page as well

Reply:
@clubkemp what I would like at least here is that, if we could move this function to utils and feed it an array or object of input. So it becomes reusable.

Reply:
Makes sense. Ok I'll get a new branch going to make this more modular.